integration/apps/rack/Gemfile

```diff
@@ -0,0 +1,29 @@
+require 'datadog/demo_env'
+
+source 'https://rubygems.org' do
+  gem 'rack'
+  gem 'dogstatsd-ruby'
+
+  gem 'ffi'
+  gem 'google-protobuf'
+
+  # Choose correct specs for demo environment
+  Datadog::DemoEnv.gem_spec('ddtrace') do |spec|
+    if !spec.empty?
+      gem 'ddtrace', *spec
+    else
+      # Install the latest prerelease by default
+      source 'http://gems.datadoghq.com/prerelease' do
+        gem 'ddtrace', '0.43.0.feature.profiling.98496'
```

Comment:
1. Yes, it is misleading in the sense that it can fall out of date, but I'm not sure we have a means of resolving the "latest" for this source otherwise.
2. Definitely, something we need to address.

Reply:
Do you know who's best to ping about #2? I don't mind getting the ball rolling on it.

Reply:
Ah, just saw your note on chat re: this being on the roadmap. Great! I was worried that it may get lost but it seems like we'll get reminded regularly ;)

Reply:
Removed this fixed version as a default; now just installs latest gem deployed. The profiling branch can change this as necessary.